chapters/data-collection.tex

```diff
@@ -1,362 +1,286 @@
-the data will be downloaded, and a ``gold master'' copy
-of the raw data should be permanently stored in a secure location.
-Then, a ``master'' copy of the data is placed into an encrypted location
-that will remain accessible on disk and backed up.
-This handling satisfies the rule of three:
-there are two on-site copies of the data and one off-site copy,
-so the data can never be lost in case of hardware failure.
-For this step, the remote location can be a variety of forms:
-the cheapest is a long-term cloud storage service
-such as Amazon Web Services or Microsoft Azure.
-Equally sufficient is a physical hard drive
-stored somewhere other than the primary work location
-(and encrypted with a service like BitLocker To Go).
-If you remain lucky, you will never have to access this copy --
-you just want to know it is out there, safe, if you need it.
-
-The copy of the raw data you are going to use
-should be handled with care.
-Since you will probably need to share it among the team,
-it should be placed in an encrypted storage location,
-although the data file itself may or may not need to be encrypted.
-Enterprise cloud solutions like Microsoft OneDrive can work as well.
-If the service satisfies your security needs,
-the raw data can be stored unencrypted here.
-Placing encrypted data (such as with VeraCrypt)
-into an unencrypted cloud storage location (such as Dropbox)
-may also satisfy this requirement for some teams,
-since this will never make the data visible to someone
-who gets access to the Dropbox,
-without the key to the file that is generated on encryption.
-\textit{The raw data file must never be placed in Dropbox unencrypted, however.}
-The way VeraCrypt works is that it creates a virtual copy
-of the unencrypted file outside of Dropbox, and lets you access that copy.
-Since you should never edit the raw data, this will not be very cumbersome,
-but the decryption and usage of the raw data is a manual process.
-With the raw data securely stored and backed up,
-you are ready to move to de-identification, data cleaning, and analysis.
+\section{Collecting Data Securely}
+Primary data collection almost always includes  \textbf{personally-identifiable information (PII)}
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Personally_Identifiable_Information_(PII)}}.
+PII must be handled with great care at all points in the data collection and management process, to comply with ethical requirements and avoid breaches of confidentiality. Access to PII must be restricted to team members granted that permission by the applicable Institutional Review Board or a data licensing agreement with a partner agency. Research teams must maintain strict protocols for data security at each stage of the process: data collection, storage, and sharing.
+
+\subsection{Secure data in the field}
+All mainstream data collection software automatically \textbf{encrypt}
+\sidenote{\textbf{Encryption:} the process of making information unreadable to 
```

Comment:
This side note defines Encryption differently than in Chapter 1. See below. Neither is incorrect, but should we be more consistent?

https://github.com/worldbank/d4di/blob/7f842e62c725e64adc82d870515e2820bc6bb3e6/chapters/handling-data.tex#L351-L353

Reply:
Let's discuss and consolidate; we should only include this as a sidenote once, CH 1 makes sense as the place to introduce. I'll wait to remove this until we've agreed on an exact plan.

Reply:
issue #343